src/store/auth/state.js

```diff
@@ -11,7 +11,11 @@ export default function () {
       updatedAt: '',
       id: '',
       authenticated: false,
-      verified: false
+      verified: false,
+      street: '',
+      city: '',
+      streetNo: '',
+      plz: ''
```

Comment:
Please use an object for the address and don' not use any abbreviations, especially not in german